<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSD300视频流检测</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_imgshow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_end">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>881</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开视频文件</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
